src/langcoder.hh

```diff
@@ -39,8 +39,9 @@ public:
 
   static quint32 findIdForLanguageCode3( std::string const & );
 
-  static QPair< quint32, quint32 > findIdsForName( QString const & );
-  static QPair< quint32, quint32 > findIdsForFilename( QString const & );
+  /// find id pairs like en-zh in dictioanry name
+  static QPair< quint32, quint32 > findLangIdPairFromStr( QString const & );
```

Comment:
findLangIdPairFromStr can be rename to findLangIdPairFromFileName,`findLangIdPairFromFilename` seems more meaningful.

Reply:
It should be `findLangIdPairFromStr`

Those uses the dictionaryName directly
![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/525d230b-e51d-46e6-b354-596464f70325)
![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/a09500a3-a86a-484c-8756-057fd8e26692)

Reply:
then FromName  should be ok.